### PR TITLE
[MIG] web_favicon: Migrated to 10.0

### PR DESCRIPTION
**Pull request by nikiwaibel**

### CODE REVIEW

File: web_favicon/views/res_company.xml [hunk 1]
```diff
@@ -5,12 +5,14 @@
...
             <field name="inherit_id" ref="base.view_company_form" />
             <field name="arch" type="xml">
-                <page name="configuration" position="inside">
-                    <group string="Favicon" name="favicon">
-                        <field name="favicon_backend" widget="image" />
-                        <field name="favicon_backend_mimetype" attrs="{'required': [('favicon_backend', '!=', False)]}" />
-                    </group>
-                </page>
+                <notebook position="inside">
+                    <page string="Web Favicon">
```
Comment (yajo) [inline web_favicon/views/res_company.xml]: add `name="favicon"` to `<page>` please

**Review by yajo — CHANGES_REQUESTED** [review]
Welcome to OCA!

Please remember to sign the CLA, and please could you take this chance to add usage instructions to the README?

Thanks!

**Comment by nikiwaibel** [comment]
@Yajo CLA is signed and sent already.

**Comment by pedrobaeza** [comment]
Can you tell me with which e-mail you sent it?

**Review by yajo — APPROVED** [review]

**Comment by nikiwaibel** [comment]
@pedrobaeza:

> # 
> 
> Date: Thu, 3 Nov 2016 02:16:37 +0100
> Message-ID: <CAEp2ira3iC6WE1=vDfPE5xm8bdw4Rwg8ToQHh+Aiw8eaKCRptw@mail.gmail.com>
> Subject: ECLA / ICLA
> From: Odoo odoo@it-meisterei.eu
> 
> # To: cla@odoo-community.org

**Comment by pedrobaeza** [comment]
CLA checked. Please change README.rst file to point to 10.0 version instead of 9.0

**Comment by pedrobaeza** [comment]
There's still at least one reference to 9.0 in the README (The "Try me on runbot" button).

**Comment by nikiwaibel** [comment]
Should that be removed, or is it OK to just change the 9.0 to 10.0?

**Comment by pedrobaeza** [comment]
You have to change it to 10.0.

**Comment by nikiwaibel** [comment]
done. sorry for rather slow progress. i am new to all this great runbot/codecov/travis automation thingies :). when is usually the main (10.0) branch updated?

**Comment by lasley** [comment]
@nikiwaibel - The Runbot branches are built automatically upon push to the associated branch on Github. In the case of 10.0, it will be rebuilt every time a new PR is merged.

**Review by lasley — CHANGES_REQUESTED** [review]
Thanks for the submission! Always great to see new faces in OCA 😄 

Please change `from openerp` imports to the new v10 namespace `from odoo`

Also, the XML files should be upgraded from the old syntax. In order to do that, you basically just remove the `<openerp>` and `<data>` tags in favor of `<odoo>`. 

Example

```
<openerp>
    <data>
        <record id="view_company_form" model="ir.ui.view">
                ...
        </record>
    </data>
</openerp>
```

Becomes

```
<odoo>
    <record id="view_company_form" model="ir.ui.view">
            ...
    </record>
</openerp>
```

**Comment by nikiwaibel** [comment]
@lasley done. thanks for explanations!

**Review by lasley — APPROVED** [review]
LGTM thanks @nikiwaibel

**Review by pedrobaeza — APPROVED** [review]
Thanks for your contribution. I hope this is the first of a lot to come! Merging.

**Pull request merged**